### DISCUSSION TITLE
SSL Connection Timeout when Running Foundryup Script

### DISCUSSION THREAD

**Discussion by hyh88384092** [foundry-rs/foundry#5775] [Help]
### Issue Description

I attempted to install the `foundryup` tool on my virtual machine using the following command:

sudo curl -L https://foundry.paradigm.xyz/ | bash

However, during the execution, I encountered the following error message:

curl: (28) SSL connection timeout

**Comment by temisan0x** [comment]
Hello @hyh88384092 have you been able to resolve this issue?

**Comment by moyinmaala** [comment]
i had the same  issue. solution is to use another network and modify firewall and proxy settings before installing foundry
